software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -190,45 +190,14 @@ public void install() {
     private ImmutableList<String> installDebianBased() {
         return ImmutableList.<String>builder()
                 .add("curl https://packagecloud.io/install/repositories/basho/riak/script.deb | " + BashCommands.sudo("bash"))
-                .add(BashCommands.sudo("apt-get install --assume-yes riak"))
+                .add(BashCommands.sudo("apt-get install --assume-yes riak=" + getEntity().getFullVersion() + "-1"))
```

Comment:
This is how packagecloud works https://packagecloud.io/basho/riak/packages/ubuntu/precise/riak_2.0.5-1_amd64.deb
https://packagecloud.io/basho/riak/packages/el/5/riak-2.0.5-1.x86_64.rpm